chart/k8gb/values.yaml

```diff
@@ -17,12 +17,11 @@ k8gb:
     enabled: false
     ip: "172.17.0.1"
     hostnames:
-     - "test-gslb-ns-us.example.com"
-     - "test-gslb-failover-ns-us.example.com"
+     - "gslb-ns-us.example.com"
   reconcileRequeueSeconds: 30
 
 externaldns:
-  image: k8s.gcr.io/external-dns/external-dns:v0.7.4
+  image: absaoss/external-dns:v0.7.1-401-gede9767c # temporary image containing NS record support before the merge of https://github.com/kubernetes-sigs/external-dns/pull/1813
   interval: "20s"
   expose53onWorkers: true # open 53/udp on workers nodes with nginx controller
```

Comment:
probably can use known port name instead of number, like `exposeUdpOnWorkers`

Reply:
```
 cat /etc/services |grep 53/udp|head
domain           53/udp     # Domain Name Server
```
53 is more clear and explicit imho. Like in Route53 naming itself lol